main.js

```diff
@@ -87,16 +81,16 @@ if (config.antiAntiAFK) setInterval(function () {
 }, 50000)
 
 function cmdInput() {
-	prompt.get("cmd", function (err, result) {
-		userInput(result.cmd, false);
+	rl.question("$ ", (cmd) => {
+		userInput(cmd, false);
 		cmdInput();
 	});
 }
 
 // function to disconnect from the server
 function stop() {
 	webserver.isInQueue = false;
-	finishedQueue = false;
+	finishedQueue = config.minecraftserver.hostname !== "2b2t.org";
```

Comment:
Maybe not hardcode this host everywhere one might want to slap another proxy in between.

Reply:
Ok I'll add an config option

Reply:
Awesome thanks!

Reply:
No problem, it is nice to see that others read my code